p/pytorch-lightning/pytorchlightning_ubi_9.3.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : pytorch-lightning
+# Version       : 2.2.4
+# Source repo : https://github.com/Lightning-AI/pytorch-lightning.git
+# Tested on     : UBI:9.3
+# Language      : Python
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Sai Kiran Nukala <sai.kiran.nukala@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on the given
+# platform using the mentioned version of the package.
+# It may not work as expected with newer versions of the
+# package and/or distribution. In such a case, please
+# contact the "Maintainer" of this script.
+#
+# -----------------------------------------------------------------------------
+
+# Exit immediately if a command exits with a non-zero status
+set -e
+
+# Variables
+PACKAGE_NAME=pytorch-lightning
+PACKAGE_VERSION=${1:-2.2.4}
+PACKAGE_URL=https://github.com/Lightning-AI/pytorch-lightning.git
+
+# Install dependencies and tools
+yum install -y git wget gcc gcc-c++ python python3-devel python3 python3-pip openblas-devel
+pip install numpy==1.26.4 wheel
+# Clone and install PyTorch
+git clone https://github.com/pytorch/pytorch.git
+cd pytorch
+wget https://repo.anaconda.com/miniconda/Miniconda3-py39_4.9.2-Linux-ppc64le.sh -O miniconda.sh
+bash miniconda.sh -b -u -p $HOME/miniconda
```

Comment:
Avoid using miniconda in our build-scripts. I think python3.9, 3.10, cmake, ninja  is available in ubi9.4. Also rust needs to download from source